app/containers/RoomTypeIcon/OmnichannelRoomIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { StyleProp, ViewStyle } from 'react-native';
+import { SvgUri } from 'react-native-svg';
+import { useSelector } from 'react-redux';
+
+import { OmnichannelSourceType, IApplicationState, IOmnichannelSourceConnected } from '../../definitions';
+import { STATUS_COLORS } from '../../constants/colors';
+import { CustomIcon } from '../../lib/Icons';
+
+const iconMap = {
+	widget: 'livechat-monochromatic',
+	email: 'mail',
+	sms: 'sms',
+	app: 'omnichannel',
+	api: 'omnichannel',
+	other: 'omnichannel'
+};
+
+interface IOmnichannelRoomIconProps {
+	size: number;
+	type: string;
+	style?: StyleProp<ViewStyle>;
+	status?: string;
+	sourceType?: IOmnichannelSourceConnected;
+}
+
+export const OmnichannelRoomIcon = ({ size, style, sourceType, status }: IOmnichannelRoomIconProps) => {
+	if (sourceType?.type === OmnichannelSourceType.APP && sourceType.id && sourceType.sidebarIcon && sourceType.connected) {
+		const baseUrl = useSelector((state: IApplicationState) => state.server?.server);
```

Comment:
You should never call hooks inside conditions.
Please read this: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

My suggestion is to create another component for the SVG.